README.md

```diff
@@ -20,3 +20,93 @@ have to start from scratch to fill it with content.
 6. Then do `npm start` and go to http://0.0.0.0:3000/
 7. To see the admin view go to http://0.0.0.0:3000/login and enter a username and
 password (provided by us).
+
+## Guide for site admins
+
+To log in as as an admin go to: http://quodl-site.herokuapp.com/login and enter
+your username and password. If you do not yet have a username and password ask
+another admin to set up an account for you.
+
+Once you have logged in you should be able to see two menus on every page:
+<img src="https://user-images.githubusercontent.com/16775804/35848171-fde190e0-0b14-11e8-8e04-78bcf8a9c287.png" width=70px />
+
+<img src="https://user-images.githubusercontent.com/16775804/35848181-071feef4-0b15-11e8-84a7-289ba432cb40.png" width=170px />
+
+Clicking on either menu opens up their options. The 'A' menu has the same options
+across all pages, whereas the options of the page menu change according to which
+page you are on.
+
+The 'A' menu is an overview of all of a given thing over the site ie. users,
+messages sent by the contact form, blog articles etc. Clicking on each category
+opens up information about that thing ie. users and gives you the ability to
+change them.
+
+Admins shouldn't need to change much using the page menu. It is powerful and
+capable of deleting and creating new pages which you do not want to do
+accidentally. There are two main cases in which you may wish to use this menu:
+
+1. If you made a change to a page and want to see what the page looked like
+previously. To do this click on 'Page Versions':
+
+<img src="https://user-images.githubusercontent.com/16775804/35848458-246cd034-0b16-11e8-84ab-8513f384ff43.png" width=150px />
+
+Here you can see recent changes to the page and revert back to them if needs be.
+Note: be careful and pay attention when reverting changes.
+
+<img src="https://user-images.githubusercontent.com/16775804/35848520-5a3dbc64-0b16-11e8-9c37-fa4301e80571.png" width=800px />
+
+2. If you want to create a new blog post. Start on the main blog page and click
```

Comment:
We might want to put this in it's own 'Creating a blog post' section as it will be the most commonly used action so a little restructuring might be in order here.

Reply:
I've rearranged it so the blog parts are in one section.